packages/fxa-settings/src/index.tsx

```diff
@@ -41,14 +57,30 @@ try {
   render(
     <React.StrictMode>
       <AppContext.Provider value={appContext}>
-        <AppErrorBoundary>
-          <App
-            {...{
-              flowQueryParams,
-              navigatorLanguages: navigator.languages,
-            }}
-          />
-        </AppErrorBoundary>
+        <FlowContext.Provider value={flowQueryParams} >
+        <AppLocalizationProvider
```

Comment:
I think I've said this in other PRs but I think your prettier settings aren't picking up our settings, this should be indented 🤔

Reply:
Oh nooo I tried to fix it but clearly didn't -- I'll fix it for good now 😂